packages/agoric-cli/src/commands/vaults.js

```diff
@@ -59,6 +77,116 @@ export const makeVaultsCommand = async logger => {
       }
     });
 
+  vaults
+    .command('info')
+    .description('show governance parameters of a vault type')
+    .requiredOption('--manager [N]', 'manager number', s => Number(s), 0)
+    .action(async ({ manager }) => {
+      const { entries, values } = Object;
+      /** @type {import('../lib/format.js').AssetDescriptor[]} */
+      // @ts-expect-error Brand vs. RpcRemote
+      const assets = values(agoricNames.vbankAsset).map(d => ({
+        ...d,
+        petname: d.issuerName,
+      }));
+      const fmtAmt = makeAmountFormatter(assets);
+      console.log(`manager${manager} governance parameters:`);
+      const { governance } = await getGovernanceState(manager);
```

Comment:
if we don't have a way to render a `.governance` node now's a good time to abstract that into a util in `lib/format.js`